aptos-move/aptos-gas-calibration/README.md

```diff
@@ -90,4 +141,18 @@ For every Calibration Function, the Abstract Gas Usage and running time are dete
 
 If the matrix that represents this system is not invertible, then we report the undetermined gas parameters, or the linearly dependent combinations of gas parameters. The exact math can be found under `/src/math.rs`. 
 
-Otherwise, the User can expect to see all the values, expressed as Gas Usage per Microsecond, along with the running times and any outliers. 
\ No newline at end of file
+Otherwise, the User can expect to see all the values, along with the running times and any outliers. 
+
+### I see "linearly dependent variables" instead of the gas costs, what do I do?
+
+If you happen to see something like:
+
+```
+linearly dependent variables are:
+
+- gas parameter: HASH_BLAKE2B_256_BASE
+- gas parameter: HASH_BLAKE2B_256_PER_BYTE
+```
+
+It is possible that you may not have enough Calibration Functions. In this example, there should be at least two Calibration Functions. 
```

Comment:
Can you list out the possibilities of why getting "linearly dependent variable"? I think sth like
"Linearly dependent variable indicates either you don't have enough samples on the specific parameter you are trying to calibrate or you have more gas parameter than you would need"? Would be better to elaborate why this could happen.

Reply:
Good point, I noted some of the possibilities. There aren't many, but I explained a couple that could happen